codefresh.yml

```diff
@@ -0,0 +1,82 @@
+# More examples of Codefresh YAML can be found at
+# https://codefresh.io/docs/docs/yaml-examples/examples/
+
+version: "1.0"
+# Stages can help you organize your steps in stages
+stages:
+  - "clone"
+  - "test"
+  - "release"
+
+steps:
+  clone:
+    title: "Cloning repository"
+    type: "git-clone"
+    repo: "a2i2/surround"
+    # CF_BRANCH value is auto set when pipeline is triggered
+    # Learn more at codefresh.io/docs/docs/codefresh-yaml/variables/
+    revision: "${{CF_BRANCH}}"
+    git: "github"
+    stage: "clone"
+
+  test:
+    title: "Run Python tests"
+    type: "freestyle" # Run any command
+    image: "python:3.6.10" # The image in which command will be executed
+    working_directory: "${{clone}}" # Running command where code cloned
+    commands:
+      # Install surround
+      - python3 setup.py install
+
+      # Run tests
+      - python3 setup.py test
+
+      # Run pylint tests
+      - pylint setup.py
+      - find surround/ -iname "*.py" | xargs pylint
+      - find examples/ -iname "*.py" | xargs pylint
+
+      # Run examples
+      - ls examples/ | xargs -n 1 -I '{}' python3 examples/'{}'/main.py
+    stage: "test"
+
+  approval:
+    title: "Pending Approval"
+    type: pending-approval
+    description: "Please approve the release to Pypi."
+    timeout:
+      duration: 2
+      finalState: approved
+    when:
+      branch:
+        only:
+          - master
+          - /([0-9]+)\.([0-9]+)\.([0-9]+)?$/
+    stage: "release"
```

Comment:
I'd be inclined to remove this. Tagging the repo with a version number should be enough of a deliberate action in this case.